polkadot-parachains/statemine/src/weights/pallet_uniques.rs

```diff
@@ -46,34 +45,37 @@ use sp_std::marker::PhantomData;
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> pallet_uniques::WeightInfo for WeightInfo<T> {
 	// Storage: Uniques Class (r:1 w:1)
+	// Storage: Uniques ClassAccount (r:0 w:1)
 	fn create() -> Weight {
-		(37_882_000 as Weight)
+		(31_145_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
-			.saturating_add(T::DbWeight::get().writes(1 as Weight))
+			.saturating_add(T::DbWeight::get().writes(2 as Weight))
```

Comment:
There is an extra write @coderobe @gilescope

Reply:
Aligned with Substrate weights: https://github.com/paritytech/substrate/blob/polkadot-v0.9.16/frame/uniques/src/weights.rs#L80